website/content/docs/connect/ca/vault.mdx

```diff
@@ -227,6 +227,11 @@ path "/sys/mounts/connect_inter" {
   capabilities = [ "read" ]
 }
 
+# Needed for Consul 1.11+
```

Comment:
I intentionally simplified this to 1.11+.

Saying 1.14.0+, 1.13.2+, 1.12.5+, and 1.11.9+ is more "technically accurate", but also more confusing and hard to follow. I think it's safest to just say "1.11+".